app/src/main/res/values-ru-rRU/strings.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<!-- Общее -->
+	<string name="app_name">SunProxy</string>
+	<string name="license" translatable="false">GPL-3.0-or-later</string>
+	<string name="home_url" translatable="false">https://github.com/UlyssesZh/SunProxy </string>
+
+	<string name="ok">ОК</string>
+	<string name="cancel">Отмена</string>
+	<string name="invalid_input">Неверный ввод</string>
+
+	<!-- Главная активность -->
+	<string name="about">%1$s %2$s\nЛицензия: %3$s\n<a href=\"%4$s\">%4$s</a></string>
```

Comment:
Use named placeholders e.g. `%{app_name}` instead of nunbered placeholders for better readability.